crates/project-model/src/build_scripts.rs

```diff
@@ -70,15 +71,25 @@ impl WorkspaceBuildScripts {
             }
         }
 
+        const RUST_1_62: Version = Version::new(1, 62, 0);
+
+        match toolchain {
+            Some(v) if v >= &RUST_1_62 => {
+                cmd.args(&["-Z", "unstable-options", "--keep-going"]).env("RUSTC_BOOTSTRAP", "1");
```

Comment:
I don't think we should use `RUSTC_BOOTSTRAP`. In addition this will need to check if `--keep-going` still exists and isn't for example removed.

Reply:
We already use `RUSTC_BOOTSTRAP` for a few different unstable cargo features we'd like to use. What would be the alternative, we can't rely on people running nightly toolchains so `+nightly` isn't an option.

Reply:
In those cases we gracefully degrade if it fails with `RUSTC_BOOTSTRAP`. In this case it would completely break a core feature if `--keep-going` doesn't work anymore.

Reply:
Ye, that is indeed a good point. Is there a simple way to detect if the command fails because of the flag not working? Retrying the non bootstrap command if hte bootstrap one fails sounds bad, given it can fail due to the buildscripts not working properly

Reply:
It gives an error like

```
error: Found argument '--foo' which wasn't expected, or isn't valid in this context

        If you tried to supply `--foo` as a value rather than a flag, use `-- --foo`

USAGE:
    cargo check [OPTIONS]

For more information try --help
```

Reply:
I guess searching the error message is the best we can do here

Reply:
Maybe we could just rerun once with `--keep-going` if the build fails? :thinking: Presumably it'd pick up where the previous build left off, and if the flag doesn't exist we'd be no worse off...

Reply:
> Maybe we could just rerun once with `--keep-going` if the build fails?

Wow, indeed!